UNIX/build.sh

```diff
@@ -61,6 +67,9 @@ if [[ "$BUILD_PACKAGE" == "Yes" ]]; then
 elif [[ "$TEST_SUITE" == "CheckedC_LLVM" ]]; then
   echo ninja -j${BUILD_CPU_COUNT}
   ninja -j${BUILD_CPU_COUNT}
+elif [[ "$CLANGD" == "Yes" ]]; then
+  echo ninja -j${BUILD_CPU_COUNT} clang llvm-size llvm-strip clangd
+  ninja -j${BUILD_CPU_COUNT}
```

Comment:
You are echoing `ninja -j${BUILD_CPU_COUNT} clang llvm-size llvm-strip clangd` but running `ninja -j${BUILD_CPU_COUNT}`. I think you would want to make both of them the same.

Reply:
Thanks for pointing this out! Fixed it.